src/server/test_utils.cc

```diff
@@ -680,6 +680,21 @@ fb2::Fiber BaseFamilyTest::ExpectConditionWithSuspension(const std::function<boo
   return fb;
 }
 
+util::fb2::Fiber BaseFamilyTest::ExpectUsedKeys(const std::vector<std::string_view>& keys) {
+  vector<LockFp> key_fps;
+  for (const auto& k : keys) {
+    key_fps.push_back(LockTable::Fingerprint(GetLockTag(k)));
+  }
+  sort(key_fps.begin(), key_fps.end());
+  auto cb = [=] {
```

Comment:
Why copy by value the vector `=`? 

It's a test so it doesn't really matter don't bother with it

Reply:
what do you mean? I have to. I move the callback to ExpectConditionWithSuspension and then return. 
If I do not capture by value, the callback will reference the unwinded stack.